Calculator/Calculator/Controller/ViewController.swift

```diff
@@ -7,12 +7,126 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    @IBOutlet weak var operatorLabel: UILabel!
+    @IBOutlet weak var operandLabel: UILabel!
+    @IBOutlet weak var stackView: UIStackView!
+    @IBOutlet weak var scrollView: UIScrollView!
+    private var expression: String = ""
+    private var isResult: Bool = false
+    private var isInputZero: Bool = true
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        clearLabel()
     }
 
+    @IBAction func touchUpAllClearButton(_ sender: UIButton) {
+        stackView.subviews.forEach { $0.removeFromSuperview() }
+        clearLabel()
+        expression = ""
+        configureCondition(isResult: false, isInputZero: true)
+    }
+    
+    @IBAction func touchUpCleanEntryButton(_ sender: UIButton) {
+        guard !isResult else {
+            return
+        }
+        
+        operandLabel.text = "0"
+        configureCondition(isResult: false, isInputZero: false)
+    }
+    
+    @IBAction func touchUpSignButton(_ sender: UIButton) {
+        guard let operand = operandLabel.text, operand != "0", operand != "0." else {
```

Comment:
`"0"`과 `"0."`을 따로 제약을 잡아 조건이 많은 것 같습니다. 다음과 같이 표현해볼 수 있을 것 같습니다.

```suggestion
        guard let operand = operandLabel.text,
              Double(operand) != Double.zero else {
```

Reply:
안내해주신 방법으로 수정하였습니다.